### ISSUE TITLE
Detect home droppable list

### ISSUE THREAD

**Issue by stefanromeis** [atlassian/react-beautiful-dnd#1399]
I face the issue, that the <Draggables> are lying behind the <Droppable>'s which are rendered after the the home list when dragging. The reason for that is, that we need apply `position: sticky` to every <Droppable>. 

Possible solution for that would be, to identify the home <Droppable> to reduce the z-index on that component.

Maybe there is a solution for that?

**Comment by alexreardon** [comment]
Can you please elaborate on your question a bit more. I am not sure what you are asking for

**Comment by stefanromeis** [comment]
Ok, let me try again. When you move a `<Draggable>` from left to the right it is hidden behind the `<Droppable>`. Right to left is overlapping fine. I've made an example: https://codesandbox.io/s/vertical-list-4hc6x. Maybe you have an easy approach to solve that without removing `position static` from the <Droppable>-containers. 

An idea would be raise the z-index on the source component and reduce them on all possible destinations. Maybe I'm thinking too complicated, but I'm out of ideas.

**Comment by stefanromeis** [comment]
Solved it. onDragStart provides the `droppableId` of the source component. So I was able raise the zIndex on the source <Droppable>. Thanks anyway :)
